### PR TITLE
bug on scheduler args changed when resume

### LINKED ISSUE #70: [BUG] lr scheduler not changed when resume and args is changed
**Describe the bug**
When resume training and the scheduler args is changed, the scheduler behavior is not changed.

**To Reproduce**
1. train model with this lr_scheduler config:
```yaml
lr_scheduler: {
    method: StepLR,
    args: {step_size: 200}
}
```
2. resume training with this config:
```yaml
lr_scheduler: {
    method: StepLR,
    args: {step_size: 350}
}
```
3. the learning rate still stepped when hit epoch 200.

**Additional context**
this happened because the lr scheduler load the state dict from the resumed model, should be changed.

### PR DESCRIPTION
**Pull request by triwahyuu**
## Type of changes

 

Please check the type of change your PR introduces:
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

close #70 

## Checklist

- [x] Unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have filled the `[Unreleased]` [Changelog](https://github.com/nodefluxio/vortex/blob/master/CHANGELOG.md)

### CODE REVIEW

**Review by alphinside — APPROVED** [review]

**Comment by triwahyuu** [comment]
anything more to add here @alphinside
